### PR TITLE
Document on exception

### PR DESCRIPTION
**Pull request by andk**
This PR is a partial answer to issue #43 and consists of four commits.

Thr first commit tries to find a wording that describes on_exception mechanism. The other three are tiny enhancements on tests.

I hope you like.

### CODE REVIEW

File: lib/Gearman/Task.pm [hunk 1]
```diff
@@ -64,6 +64,15 @@ process.
 
 =item
 
+on_exception
+
+A subroutine reference to be invoked when the task fails. The client
+needs to be configured to support exception handling by setting the
+attribute C<exception> to a true value. The subroutine will be passed
```
Comment (p-alik) [inline lib/Gearman/Task.pm]: > attribute `C<exception>`

The attribute name is `exceptions`
https://github.com/p-alik/perl-Gearman/blob/4f221ba9a18dd52de9c54f1fc8731b3e614e743c/lib/Gearman/Client.pm#L71

**Comment by andk** [comment]
Sorry! Amended now and pushed again.

**Review by p-alik — APPROVED** [review]

**Comment by esabol** [comment]
Looks good to me!

**Comment by p-alik** [comment]
Thank you, @andk!

**Pull request merged**